src/components/tests/Calculator.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Kindly avoid using comments to skip linter errors, as it is not a good practice. Sometimes linter error happens due to syntax error or anything related to the code. So try to fix it manually and this way you may learn a new concept.

Reply:
@Tufahel : Thanks for your kind review please help me to fix it. I found this error on that line
'react-test-renderer' should be listed in the project's dependencies. Run 'npm i -S react-test-renderer' to add iteslint[import/no-extraneous-dependencies](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-extraneous-dependencies.md) 
and I already have react-test-renderer installed and in the dependencies list. You can also see it in the screenshot 
![fix](https://user-images.githubusercontent.com/112868013/236041432-a734bfda-753b-418b-a0b5-80107be58ac6.JPG)